data/rohit190183107010.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Rohiit Yaddav",
+  "type": "personal",
+  "bio": "Highly Skilled TechOps Engineer at Datahash",
+  "avatar": "https://github.com/rohit190183107010.png",
```

Comment:
Hi @rohit190183107010, thank you for contributing  :tada: 
You can delete this line, it is not required anymore :+1: 
```suggestion
```